cmake/SetupCompilerOptions.cmake

```diff
@@ -111,6 +111,20 @@ if(BLT_DEFINES)
     message(STATUS "Added \"${BLT_DEFINES}\" to definitions")
 endif()
 
+if(COMPILER_FAMILY_IS_MSVC)
+    add_definitions(-D_USE_MATH_DEFINES)
```

Comment:
Yes -- since you need this define if you are using the math headers, I think it makes sense for projects using the math headers to provide a solution instead of blt always adding it.
For axom, that would be in our generated config.hpp.